packages/core/workflows-sdk/src/helper/workflow-export.ts

```diff
@@ -527,7 +527,7 @@ function attachOnFinishReleaseEvents(
         )
         .join(EOL + separator + EOL)
 
-      logger.error(allWorkflowErrors)
+      if (allWorkflowErrors) logger.error(allWorkflowErrors)
```

Comment:
```suggestion
      if (allWorkflowErrors) {
        logger.error(allWorkflowErrors)
      }
```

Reply:
Should we maybe just commit this ourselves and get it merged?

Reply:
Let's merge it, I'll reformat in a separate PR

Reply:
Hi, sorry i've seen it only now and accepted the commit suggestion